tests/integration/database/test_db.py

```diff
@@ -768,3 +768,28 @@ def test_faceted_builder_queries(
             count=True,
         )
         assert db_records == 1
+
+    def test_get_geo_from_string(
+        self, interface, named_location_us, named_location_stoneham
+    ):
+        stoneham = Locations(
+            **{
+                "id": "34333",
+                "type": "us_postalcode",
+                "name": "2180",
+                "display_name": "Stoneham, MA (02180)",
+                "the_geom": "0103000020E61000000100000005000000BA6B09F9A0C751C046B6F3FDD4384540BA6B09F9A0C751C08E06F016484045401B9E5E29CBC451C08E06F016484045401B9E5E29CBC451C046B6F3FDD4384540BA6B09F9A0C751C046B6F3FDD4384540",  # noqa E501
+                "type_order": "4",
+            }
+        )
+        interface.db.add(stoneham)
+        interface.db.commit()
+
+        geojson_str = interface.get_geo_from_string("United States")
+        assert geojson_str == named_location_us
```

Comment:
would be nice to have some comments here on the assertions

Reply:
I got you, added.